alpha/pom.xml

```diff
@@ -75,20 +72,42 @@
       <dependencies>
         <dependency>
           <groupId>io.kamon</groupId>
-          <artifactId>kamon-log-reporter_2.12</artifactId>
+          <artifactId>kamon-autoweave_2.11</artifactId>
+          <version>${kamon.version}</version>
+        </dependency>
+        <dependency>
+          <groupId>org.aspectj</groupId>
+          <artifactId>aspectjweaver</artifactId>
         </dependency>
         <dependency>
           <groupId>io.kamon</groupId>
-          <artifactId>kamon-statsd_2.12</artifactId>
+          <artifactId>kamon-log-reporter_2.11</artifactId>
+          <version>${kamon.version}</version>
         </dependency>
         <dependency>
           <groupId>io.kamon</groupId>
-          <artifactId>kamon-autoweave_2.12</artifactId>
-          <version>0.6.5</version>
+          <artifactId>kamon-statsd_2.11</artifactId>
+          <version>${kamon.version}</version>
         </dependency>
         <dependency>
-          <groupId>org.aspectj</groupId>
-          <artifactId>aspectjweaver</artifactId>
+          <groupId>com.typesafe.akka</groupId>
+          <artifactId>akka-actor_2.11</artifactId>
```

Comment:
Current Alpha Server doesn't use Akka, why did you put these dependencies.

Reply:
saga-core-akka support is there right so i thought akka support is there. i will remove unwated dependency.